Testing/xASL_test_UnitTesting.m

```diff
@@ -89,9 +99,9 @@
     
     %% Print test results
     clc
-    fprintf('================================= TEST RESULTS =================================\n\n')
+    fprintf('====================================== TEST RESULTS ======================================\n\n')
     disp(UnitTestsTable);
```

Comment:
Replace `disp` or `display` by `fprintf('%s\n', MessageToPrint);`

Reply:
But the UnitTestsTable isn't a string, it's a table. You can't fprintf a table.